### PR TITLE
ISO: enable CONFIG_EXT4_FS_SECURITY in arm64

### PR DESCRIPTION
**Pull request by spowelljr**
Fixes https://github.com/kubernetes/minikube/issues/18738

**Before:**
```
$ minikube start --driver qemu
...
$ minikube addons enable ingress
...
$ kubectl get pods -A
NAMESPACE       NAME                                        READY   STATUS             RESTARTS      AGE
ingress-nginx   ingress-nginx-admission-create-lntsx        0/1     Completed          0             51s
ingress-nginx   ingress-nginx-admission-patch-54kvv         0/1     Completed          0             51s
ingress-nginx   ingress-nginx-controller-768f948f8f-tzrpd   0/1     ImagePullBackOff   0             51s
```

**After:**
```
$ minikube start --driver qemu
...
$ minikube addons enable ingress
...
$ kubectl get pods -A
NAMESPACE       NAME                                        READY   STATUS      RESTARTS   AGE
ingress-nginx   ingress-nginx-admission-create-rq64x        0/1     Completed   0          22s
ingress-nginx   ingress-nginx-admission-patch-p8nnn         0/1     Completed   0          22s
ingress-nginx   ingress-nginx-controller-768f948f8f-w5vng   0/1     Running     0          22s
```

### CODE REVIEW

**Comment by spowelljr** [comment]
ok-to-build-iso

File: deploy/iso/minikube-iso/board/minikube/aarch64/linux_aarch64_defconfig [hunk 1]
```diff
@@ -1283,3 +1283,4 @@ CONFIG_IP_ADVANCED_ROUTER=y
 CONFIG_IP_ROUTE_MULTIPATH=y
 CONFIG_IP_MULTICAST=y
 CONFIG_IP_MULTIPLE_TABLES=y
+CONFIG_EXT4_FS_SECURITY=y
```
Comment (medyagh) [inline deploy/iso/minikube-iso/board/minikube/aarch64/linux_aarch64_defconfig:1286]: we might have to enable 
CONFIG_SECURITY_FILE_CAPABILITIES
to be able to enable this

Comment (spowelljr) [reply deploy/iso/minikube-iso/board/minikube/aarch64/linux_aarch64_defconfig:1286]: Not needed, also not included with x86

**Review by medyagh — APPROVED** [review]

**Pull request merged**